### PR TITLE
Make the benchmarks app runnable locally

### PR DESCRIPTION
**Pull request by davidfowl**
- Detect BenchmarksTargetFramework and change project behavior based on it being set

### CODE REVIEW

File: build/dependencies.props [hunk 1]
```diff
@@ -42,4 +39,14 @@
...
   </PropertyGroup>
+
+  <PropertyGroup Condition="$(TargetFramework) == 'netcoreapp2.1'">
+    <MicrosoftAspNetCoreAppPackageVersion>2.1.3</MicrosoftAspNetCoreAppPackageVersion>
+  </PropertyGroup>
+  <PropertyGroup Condition="$(TargetFramework) == 'netcoreapp2.2'">
+    <MicrosoftAspNetCoreAppPackageVersion>2.2.0</MicrosoftAspNetCoreAppPackageVersion>
+  </PropertyGroup>
+  <PropertyGroup Condition="$(TargetFramework) == 'netcoreapp3.0'">
+    <MicrosoftAspNetCoreAppPackageVersion>3.0.0-preview7.19351.4</MicrosoftAspNetCoreAppPackageVersion>
```
Comment (davidfowl) [inline build/dependencies.props]: Ideally maestro would update this.

Comment (BrennanConroy) [reply build/dependencies.props]: We can do that probably

File: build/dependencies.props [hunk 2]
```diff
@@ -42,4 +39,14 @@
     
     <!-- Using 2.2 EF version for netcoreapp3.0 benchmarks as it doesn't work with current providers -->
   </PropertyGroup>
+
+  <PropertyGroup Condition="$(TargetFramework) == 'netcoreapp2.1'">
+    <MicrosoftAspNetCoreAppPackageVersion>2.1.3</MicrosoftAspNetCoreAppPackageVersion>
```
Comment (BrennanConroy) [inline build/dependencies.props]: While you're here, `2.1.11`?

File: build/dependencies.props [hunk 3]
```diff
@@ -42,4 +39,14 @@
...
+  </PropertyGroup>
+  <PropertyGroup Condition="$(TargetFramework) == 'netcoreapp2.2'">
+    <MicrosoftAspNetCoreAppPackageVersion>2.2.0</MicrosoftAspNetCoreAppPackageVersion>
```
Comment (BrennanConroy) [inline build/dependencies.props]: And `2.2.5`?

**Comment by davidfowl** [comment]
@sebastienros can you take a look?

**Comment by sebastienros** [comment]
I tried to run this branch with all tfms on the infra and it worked well (once I merged master).

**Pull request merged**